src/quo2/components/list_items/account_list_card/view.cljs

```diff
@@ -0,0 +1,36 @@
+(ns quo2.components.list-items.account-list-card.view
+  (:require
+    [quo2.components.avatars.account-avatar.view :as account-avatar]
+    [quo2.components.icon :as icon]
+    [quo2.components.markdown.text :as text]
+    [quo2.foundations.colors :as colors]
+    [quo2.theme :as quo.theme]
+    [react-native.core :as rn]
+    [quo2.components.list-items.account-list-card.style :as style]))
+
+(defn- internal-view
+  [{:keys [theme account-props network state action]}]
+  [rn/view
+   {:style               (style/container state theme)
+    :accessibility-label :container}
+   [rn/view
+    {:style {:flex-direction :row
+             :align-items    :center}}
+    [account-avatar/view account-props]
+    [rn/view {:style {:margin-left 8}}
+     [text/text
+      {:weight :semi-bold
+       :size   :paragraph-2} (:name account-props)]
+     [text/text
+      {:size  :paragraph-2
+       :style {:color (get colors/networks network)}} "eth:"
```

Comment:
1. shouldn't `"eth:"` be in i18n resources
2. can it be in the next line? Visually it looks as a part of styles map atm

Reply:
@rasom do you mean a translation? I don't think `eth` in translatable

Reply:
Yeah but should it be a prop or something or the value is always eth:?

Reply:
In the design walkthrough, Pedro once said that the `eth:`prefix is needed for multichain addresses (`eth:opt:arb:0x3s...`). For the legacy chain, it's not needed. We should check with the design team.

Reply:
Fixed it!